app/controllers/admin/orders_controller.rb

```diff
@@ -2,6 +2,15 @@ class Admin::OrdersController < ApplicationController
   layout "admin"
 
   def index
+    @orders = Order.all
+    @orders = Order.ordered if params[:status] == '0'
+    @orders = Order.paid if params[:status] == '1'
+    @orders = Order.completed if params[:status] == '2'
+    @orders = Order.cancelled if params[:status] == '3'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.